### PR TITLE
Before apply, check in use and show prompt

### PR DESCRIPTION
**Pull request by gabriel**
This is a prototype based on my comment here:
https://keybase.atlassian.net/wiki/display/SPEC/Updater+design+document?focusedCommentId=7766037#comment-7766037

This adds a generic prompt to the Updater (UI) app.

### CODE REVIEW

**Comment by gabriel** [comment]
2016-04-28 at 4 53 pm

**Comment by gabriel** [comment]
@keybase/updater-hackers

File: keybase/prompt.go [hunk 1]
```diff
@@ -65,3 +65,37 @@ func (c context) updatePrompt(promptCommand string, update updater.Update, optio
...
+		Message: "You have files, folders or a terminal open in Keybase.\n\nYou can force the update. That would be like yanking a USB drive and plugging it right back in. It'll instantly give you the latest version of Keybase, but you'll need to reopen any files you're working with. If you're working in the terminal, you'll need to cd out of /keybase and back in.",
+		Buttons: []string{"Force update", "Try again later"},
+	})
+	if err != nil {
+		return fmt.Errorf("Error generating input: %s", err)
+	}
+
+	var result struct {
+		Button string `json:"button"`
+	}
```
Comment (jzila) [inline keybase/prompt.go]: Please create an actual type for this outside this method's scope.

**Comment by jzila** [comment]
Go looks fine to me other than one nit. Need someone else to look at the objective C though.

**Pull request merged**